### PR TITLE
Fix for running iOS app on Xcode 12.5

### PR DESCRIPTION
**Pull request by MarkDavies0**
## Changes
Fixes a [build error when running on Xcode 12.5](https://github.com/facebook/flipper/issues/2215#issuecomment-829053571)

- Updates the Flipper pod
- Bumps up the min iOS version to iOS 13

## Requirements
Requires: Xcode updated to 12.5. Delete Pods and run `pod install --repo-update` in ios directory.

## PR Dependencies
Depends on: #37.

## Merged Branches and Other Dependencies
Merged branch(es): None
Other dependencies: None

## Subtasks / Progress
N/A

## Preview
N/A

## Testing
Manual

### CODE REVIEW

File: mobile/ios/Podfile [hunk 1]
```diff
@@ -15,15 +15,35 @@ target 'CompanionKit' do
...
-  # Enables Flipper.
-  #
-  # Note that if you have use_frameworks! enabled, Flipper will not work and
-  # you should disable these next few lines.
-  use_flipper!
+  use_flipper!({ 'Flipper-Folly' => '2.5.3', 'Flipper' => '0.87.0', 'Flipper-RSocket' => '1.3.1' })
+
+  # Fix for Folly Pod on Xcode 12.5
+
+  def find_and_replace(dir, findstr, replacestr)
```
Comment (rlnsy) [inline mobile/ios/Podfile:22]: @MarkDavies0  just curious - was most of this auto-generated or did you write it yourself? Make sure to link the source if you're using a known fix e.g. stack overflow.

Comment (MarkDavies0) [reply mobile/ios/Podfile:22]: It comes from a response to an issue on the Facebook/flipper repo which is what I linked to above. Is there a certain style for code attribution comments in the actual file or should I just add the link?

Comment (rlnsy) [reply mobile/ios/Podfile:23]: Yeah I think just a commented link would be good

**Comment by rlnsy** [comment]
@MarkDavies0 I'm gonna test this once I upgrade Xcode to see if I can resolve the constant lock file changes.

**Comment by rlnsy** [comment]
@MarkDavies0 I updated the lock file in the base branch, so you may want to regenerate it here. I'm still working on getting my xcode updated unfortunately, but I'll update you when I've got it.

**Comment by MarkDavies0** [comment]
@rlnsy Can you try running `pod install --repo-update` and see what it does to the lock file. I have updated the requirements section of this PR

**Comment by rlnsy** [comment]
@MarkDavies0 Still running <12.5 btw, so this is just fyi. After fresh-cloning the repo and running `setup.bash` (which includes pods install with `repo-update`), I end up with the following diff:
```
diff --git a/mobile/ios/Podfile.lock b/mobile/ios/Podfile.lock
index b3a5ab0..45c7a72 100644
--- a/mobile/ios/Podfile.lock
+++ b/mobile/ios/Podfile.lock
@@ -23,7 +23,7 @@ PODS:
     - UMFileSystemInterface
     - UMImageLoaderInterface
     - UMPermissionsInterface
-  - EXConstants (10.1.3):
+  - EXConstants (9.1.1):
     - UMConstantsInterface
     - UMCore
   - EXCrypto (8.2.1):
@@ -470,7 +470,7 @@ DEPENDENCIES:
   - EXAppleAuthentication (from `../node_modules/expo-apple-authentication/ios`)
   - EXAV (from `../node_modules/expo-av/ios`)
   - EXCamera (from `../node_modules/expo-camera/ios`)
-  - EXConstants (from `../node_modules/sentry-expo/node_modules/expo-constants/ios`)
+  - EXConstants (from `../node_modules/expo-constants/ios`)
   - EXCrypto (from `../node_modules/expo-crypto/ios`)
   - EXDevice (from `../node_modules/expo-device/ios`)
   - EXErrorRecovery (from `../node_modules/expo-error-recovery/ios`)
@@ -597,7 +597,7 @@ EXTERNAL SOURCES:
   EXCamera:
     :path: "../node_modules/expo-camera/ios"
   EXConstants:
-    :path: "../node_modules/sentry-expo/node_modules/expo-constants/ios"
+    :path: "../node_modules/expo-constants/ios"
   EXCrypto:
     :path: "../node_modules/expo-crypto/ios"
   EXDevice:
@@ -732,31 +732,31 @@ SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
   DoubleConversion: cde416483dac037923206447da6e1454df403714
-  EXAppAuth: 854e1fbdd37333eac488ee31de6756ca235eb3f5
-  EXAppleAuthentication: 3447f90462a9563792ba3e15d11758b5db635448
-  EXAV: b50e42821a48110add008bf83f480ce6ac2b8471
-  EXCamera: d2caf6bdc821a316d070fe979a255dd5b32d2daf
-  EXConstants: c00cd53a17a65b2e53ddb3890e4e74d3418e406e
-  EXCrypto: 84a96afc7bb083729d86b934a5e1633d67a3ddc0
-  EXDevice: 841e1078d98529b176119fecfa524faf5e9d3379
-  EXErrorRecovery: a9ef867e1d5479a7c08279be84c5164d38647e08
-  EXFileSystem: 8d1dc2924d62046275c84d70b88fa3aa7daeb116
-  EXFont: 5c464ce400b92f87f907d68093c29946177295bf
-  EXGL: 83453233aaba23a1511ec9ea9d8eada517aa8c5e
-  EXGL_CPP: 6c376b940e21db279e5cd8ffb620c4d86ebffb0b
-  EXGoogleSignIn: b2dc0637bf4398ed3054d0b5d665f39f89df735d
-  EXHaptics: 0f3f8dad39646e8c0108499ad9f4924076cc4d3d
-  EXImageLoader: 8366cb1a7d207a241a1bfaa4c5ab4859cf72d507
-  EXImageManipulator: a03933ec19825a7a894cdabef9e227ce8a007f99
-  EXImagePicker: 0ddacf50173316ecb1770725a831401ea17f36dc
-  EXKeepAwake: 1c77f2d31eafaea3410c00957669f306f340b843
-  EXLinearGradient: d25940ede1e798882f57715cff9edb4075d95e2d
-  EXLocation: c50022a5a829bad7a621d42eaecc60e171d8efa7
-  EXMediaLibrary: ec1333a64018a83a0285643d133cb7adff43f7f1
-  EXPermissions: 9804d7ec7399e7b24e3d9dea2b3db976a7f6b5de
-  EXSplashScreen: 2429477a39b51c9c9433d2480cbae699e6a0374b
-  EXSQLite: b001ba35deb3001bd6e4b256581a558ae745ef2f
-  EXUpdates: 8c4570807f9435337769850d9976d63a407cecfc
+  EXAppAuth: 98c853968adeec68008887a0cb475efb2656f196
+  EXAppleAuthentication: 5b3da71bada29e2423d8ea27e5538ef0d75aba62
+  EXAV: e9c3a5e6fc4137ddb65a7d5810efc95261365ee2
+  EXCamera: efef30c8f5016b46fa3119a0c685cf75b841f453
+  EXConstants: f907b3b6ce16e20d1750f22af1e095e924574bcb
+  EXCrypto: 867e1d45d513e39ab9d7706232435edb1cf756cd
+  EXDevice: befd3b2619915e615c3753c9f83309808aa0cdf5
+  EXErrorRecovery: ff58873bf2714fcd5290935d49b74d9fc589d575
+  EXFileSystem: 76875135b61708b9afa7e6a89b72a60ba0fdfa20
+  EXFont: e583875d5d37a9e272d1cdba7f383bd576952919
+  EXGL: 5ab2299d0bae290b81ca24fc70e8f1697affbef5
+  EXGL_CPP: 03b30fb5012eee6f4231b6667d97256716e7e13b
+  EXGoogleSignIn: 02f0db1efb39c8e2508832a0175319f0ce08a80f
+  EXHaptics: 5428b344a216ca5d9df6ca8f65720b2a1ad9f109
+  EXImageLoader: 02ca02c9cd5cc8a97b423207a73a791e0a86bea5
+  EXImageManipulator: fbc0fff59cd398ebec3f3e9bedd4451375de2d5f
+  EXImagePicker: 36f2bba7d6367ecb2b1340c5db5a676b3b19a26a
+  EXKeepAwake: 8b0f68242f036b971f9f8976341823cbe6f50812
+  EXLinearGradient: 663bf3b9009a96567e98498016e40422631be81d
+  EXLocation: e93020e255f52a1cc031e0f45b0a24eb20fc4b55
+  EXMediaLibrary: 73b7e171c26e828dfadf1fd270a4498cc1902a72
+  EXPermissions: 9daf039546113d8f2f0a62db1c75d2d9206e5e4e
+  EXSplashScreen: 9423d258b71afa5bf128a83dcb57b636d9900a74
+  EXSQLite: 64611c1b369c0f1e9cae4e551d0aa4478a58cadf
+  EXUpdates: f3c4943abb6ab28cc55cbeac4ff4612701f82725
   FBLazyVector: 3ef4a7f62e7db01092f9d517d2ebc0d0677c4a37
   FBReactNativeSpec: dc7fa9088f0f2a998503a352b0554d69a4391c5a
   Firebase: 09fb40287b6dfc8ee65f726fa0b788719d3f2a07
@@ -791,7 +791,7 @@ SPEC CHECKSUMS:
   React-jsi: 54245e1d5f4b690dec614a73a3795964eeef13a8
   React-jsiexecutor: 8ca588cc921e70590820ce72b8789b02c67cce38
   React-jsinspector: b14e62ebe7a66e9231e9581279909f2fc3db6606
-  react-native-appearance: fc2014516054585d531e07aa0b40ab0de1d2be85
+  react-native-appearance: 0f0e5fc2fcef70e03d48c8fe6b00b9158c2ba8aa
   React-RCTActionSheet: 910163b6b09685a35c4ebbc52b66d1bfbbe39fc5
   React-RCTAnimation: 9a883bbe1e9d2e158d4fb53765ed64c8dc2200c6
   React-RCTBlob: 39cf0ece1927996c4466510e25d2105f67010e13
@@ -802,24 +802,24 @@ SPEC CHECKSUMS:
   React-RCTText: 1b6773e776e4b33f90468c20fe3b16ca3e224bb8
   React-RCTVibration: 4d2e726957f4087449739b595f107c0d4b6c2d2d
   ReactCommon: a0a1edbebcac5e91338371b72ffc66aa822792ce
-  RNDateTimePicker: 3a1783e63fcb6e2b162b08a4321db030d23accff
+  RNDateTimePicker: a84d2b439666c9243d42a8396c588d4d1d36e3f3
   RNFBApp: 43a062dc6a1a88edd934d2229d1ba64f25381598
   RNSentry: 6b46b6fc1d715a378fbaa5d7d43bc9ce99b500e5
   RNSVG: ce9d996113475209013317e48b05c21ee988d42e
   Sentry: 388c9dc093b2fd3a264466a5c5b21e25959610a9
-  UMAppLoader: 98b77b58fac12abab963f79f762ceef008c50505
-  UMBarCodeScannerInterface: e1299aafa3bede8ba674e84199e9f319a0195cad
-  UMCameraInterface: 68da359d6fe7423d7b114b96ea0df268a84da39f
-  UMConstantsInterface: 2569e83b7f70a0844737e1f6c2dfd1bc0c8a34d6
-  UMCore: ef0c10289950b51d0c1dae75c37fba116f2327ac
-  UMFaceDetectorInterface: b0f3be22a524053f29d71c50bfdf51dc9553f7aa
-  UMFileSystemInterface: 29a555288008c9fa9be7d781c84f58ef51d41de6
-  UMFontInterface: 5ece8ea0e3c5753a87a558dcd7e79dcf38a6cab6
-  UMImageLoaderInterface: c440e3cd54c060cb2f9a278a923e28c6fe0ec116
-  UMPermissionsInterface: d6be0aea537754c92648e3d7100c58dd72c01142
-  UMReactNativeAdapter: 5973dd1e4374cd1db4549b6bc714ccbbde9615c0
-  UMSensorsInterface: 7bc27058e00af99695e16f74c092582698be964d
-  UMTaskManagerInterface: 8b337c26b828b3e50d26776966981f4cb2ee76e4
+  UMAppLoader: 61049c8d55590b74e9ae1d5429bf68d96b4a2528
+  UMBarCodeScannerInterface: e5e4c87797d3d01214e25cd1618866caf5d4f17f
+  UMCameraInterface: 415ac060034edecacdbbaa739c223e3f276e0056
+  UMConstantsInterface: 1a52f2d884c95e8829439da13e36b7669a1a8fb4
+  UMCore: d98083b522b08c0a8ba3992bc263c624ae5d887c
+  UMFaceDetectorInterface: 67c6c82451338da01a4bc00ec46365a2a8ea9057
+  UMFileSystemInterface: 303d696ede28102a7e11d111808bd2ed2c5eb62f
+  UMFontInterface: 6edf1ee8bc55d2030766f8cf0a7b20a5d5a913b0
+  UMImageLoaderInterface: 9cdbf3bab6a513bddd88505cb2340fe02d6a11c0
+  UMPermissionsInterface: 019170ad655f464e3f8d23d2a8bcbda2e645cde4
+  UMReactNativeAdapter: 538efe92e781b5d7678cf95b34c46f2d0989a557
+  UMSensorsInterface: cb5bf31d52c4349f0ff9e3c049bbe4df0d80d383
+  UMTaskManagerInterface: 80653f25c55d9e6d79d6a0a65589fa213feaee11
   Yoga: 7740b94929bbacbddda59bf115b5317e9a161598
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a

diff --git a/mobile/yarn.lock b/mobile/yarn.lock
index 836821f..fef708c 100644
--- a/mobile/yarn.lock
+++ b/mobile/yarn.lock
@@ -9344,6 +9344,11 @@ rimraf@~2.2.6:
   resolved "https://registry.yarnpkg.com/rimraf/-/rimraf-2.2.8.tgz#e439be2aaee327321952730f99a8929e4fc50582"
   integrity sha1-5Dm+Kq7jJzIZUnMPmaiSnk/FBYI=

+rn-apple-healthkit@0.8.0:
+  version "0.8.0"
+  resolved "https://registry.yarnpkg.com/rn-apple-healthkit/-/rn-apple-healthkit-0.8.0.tgz#83a23e71fc54d6d929cd6a55d5c590072bab4c64"
+  integrity sha512-LymIWk9lwIkcGIACveCxvp2jKXTTrDbdUYgcac+wuaSLPPpLfZqrQ0fz+QACYx5cSwAn6Uj6NcJ5+QkwPjY+5g==
+
 rsvp@^4.8.4:
   version "4.8.5"
   resolved "https://registry.yarnpkg.com/rsvp/-/rsvp-4.8.5.tgz#c8f155311d167f68f21e168df71ec5b083113734"
```
Running `pod install --repo-update` again yields no more differences.

**Comment by rlnsy** [comment]
After merge there are no diffs in `yarn.lock` but this is the pods one:
```
diff --git a/mobile/ios/Podfile.lock b/mobile/ios/Podfile.lock
index b3a5ab0..45c7a72 100644
--- a/mobile/ios/Podfile.lock
+++ b/mobile/ios/Podfile.lock
@@ -23,7 +23,7 @@ PODS:
     - UMFileSystemInterface
     - UMImageLoaderInterface
     - UMPermissionsInterface
-  - EXConstants (10.1.3):
+  - EXConstants (9.1.1):
     - UMConstantsInterface
     - UMCore
   - EXCrypto (8.2.1):
@@ -470,7 +470,7 @@ DEPENDENCIES:
   - EXAppleAuthentication (from `../node_modules/expo-apple-authentication/ios`)
   - EXAV (from `../node_modules/expo-av/ios`)
   - EXCamera (from `../node_modules/expo-camera/ios`)
-  - EXConstants (from `../node_modules/sentry-expo/node_modules/expo-constants/ios`)
+  - EXConstants (from `../node_modules/expo-constants/ios`)
   - EXCrypto (from `../node_modules/expo-crypto/ios`)
   - EXDevice (from `../node_modules/expo-device/ios`)
   - EXErrorRecovery (from `../node_modules/expo-error-recovery/ios`)
@@ -597,7 +597,7 @@ EXTERNAL SOURCES:
   EXCamera:
     :path: "../node_modules/expo-camera/ios"
   EXConstants:
-    :path: "../node_modules/sentry-expo/node_modules/expo-constants/ios"
+    :path: "../node_modules/expo-constants/ios"
   EXCrypto:
     :path: "../node_modules/expo-crypto/ios"
   EXDevice:
@@ -732,31 +732,31 @@ SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
   DoubleConversion: cde416483dac037923206447da6e1454df403714
-  EXAppAuth: 854e1fbdd37333eac488ee31de6756ca235eb3f5
-  EXAppleAuthentication: 3447f90462a9563792ba3e15d11758b5db635448
-  EXAV: b50e42821a48110add008bf83f480ce6ac2b8471
-  EXCamera: d2caf6bdc821a316d070fe979a255dd5b32d2daf
-  EXConstants: c00cd53a17a65b2e53ddb3890e4e74d3418e406e
-  EXCrypto: 84a96afc7bb083729d86b934a5e1633d67a3ddc0
-  EXDevice: 841e1078d98529b176119fecfa524faf5e9d3379
-  EXErrorRecovery: a9ef867e1d5479a7c08279be84c5164d38647e08
-  EXFileSystem: 8d1dc2924d62046275c84d70b88fa3aa7daeb116
-  EXFont: 5c464ce400b92f87f907d68093c29946177295bf
-  EXGL: 83453233aaba23a1511ec9ea9d8eada517aa8c5e
-  EXGL_CPP: 6c376b940e21db279e5cd8ffb620c4d86ebffb0b
-  EXGoogleSignIn: b2dc0637bf4398ed3054d0b5d665f39f89df735d
-  EXHaptics: 0f3f8dad39646e8c0108499ad9f4924076cc4d3d
-  EXImageLoader: 8366cb1a7d207a241a1bfaa4c5ab4859cf72d507
-  EXImageManipulator: a03933ec19825a7a894cdabef9e227ce8a007f99
-  EXImagePicker: 0ddacf50173316ecb1770725a831401ea17f36dc
-  EXKeepAwake: 1c77f2d31eafaea3410c00957669f306f340b843
-  EXLinearGradient: d25940ede1e798882f57715cff9edb4075d95e2d
-  EXLocation: c50022a5a829bad7a621d42eaecc60e171d8efa7
-  EXMediaLibrary: ec1333a64018a83a0285643d133cb7adff43f7f1
-  EXPermissions: 9804d7ec7399e7b24e3d9dea2b3db976a7f6b5de
-  EXSplashScreen: 2429477a39b51c9c9433d2480cbae699e6a0374b
-  EXSQLite: b001ba35deb3001bd6e4b256581a558ae745ef2f
-  EXUpdates: 8c4570807f9435337769850d9976d63a407cecfc
+  EXAppAuth: 98c853968adeec68008887a0cb475efb2656f196
+  EXAppleAuthentication: 5b3da71bada29e2423d8ea27e5538ef0d75aba62
+  EXAV: e9c3a5e6fc4137ddb65a7d5810efc95261365ee2
+  EXCamera: efef30c8f5016b46fa3119a0c685cf75b841f453
+  EXConstants: f907b3b6ce16e20d1750f22af1e095e924574bcb
+  EXCrypto: 867e1d45d513e39ab9d7706232435edb1cf756cd
+  EXDevice: befd3b2619915e615c3753c9f83309808aa0cdf5
+  EXErrorRecovery: ff58873bf2714fcd5290935d49b74d9fc589d575
+  EXFileSystem: 76875135b61708b9afa7e6a89b72a60ba0fdfa20
+  EXFont: e583875d5d37a9e272d1cdba7f383bd576952919
+  EXGL: 5ab2299d0bae290b81ca24fc70e8f1697affbef5
+  EXGL_CPP: 03b30fb5012eee6f4231b6667d97256716e7e13b
+  EXGoogleSignIn: 02f0db1efb39c8e2508832a0175319f0ce08a80f
+  EXHaptics: 5428b344a216ca5d9df6ca8f65720b2a1ad9f109
+  EXImageLoader: 02ca02c9cd5cc8a97b423207a73a791e0a86bea5
+  EXImageManipulator: fbc0fff59cd398ebec3f3e9bedd4451375de2d5f
+  EXImagePicker: 36f2bba7d6367ecb2b1340c5db5a676b3b19a26a
+  EXKeepAwake: 8b0f68242f036b971f9f8976341823cbe6f50812
+  EXLinearGradient: 663bf3b9009a96567e98498016e40422631be81d
+  EXLocation: e93020e255f52a1cc031e0f45b0a24eb20fc4b55
+  EXMediaLibrary: 73b7e171c26e828dfadf1fd270a4498cc1902a72
+  EXPermissions: 9daf039546113d8f2f0a62db1c75d2d9206e5e4e
+  EXSplashScreen: 9423d258b71afa5bf128a83dcb57b636d9900a74
+  EXSQLite: 64611c1b369c0f1e9cae4e551d0aa4478a58cadf
+  EXUpdates: f3c4943abb6ab28cc55cbeac4ff4612701f82725
   FBLazyVector: 3ef4a7f62e7db01092f9d517d2ebc0d0677c4a37
   FBReactNativeSpec: dc7fa9088f0f2a998503a352b0554d69a4391c5a
   Firebase: 09fb40287b6dfc8ee65f726fa0b788719d3f2a07
@@ -791,7 +791,7 @@ SPEC CHECKSUMS:
   React-jsi: 54245e1d5f4b690dec614a73a3795964eeef13a8
   React-jsiexecutor: 8ca588cc921e70590820ce72b8789b02c67cce38
   React-jsinspector: b14e62ebe7a66e9231e9581279909f2fc3db6606
-  react-native-appearance: fc2014516054585d531e07aa0b40ab0de1d2be85
+  react-native-appearance: 0f0e5fc2fcef70e03d48c8fe6b00b9158c2ba8aa
   React-RCTActionSheet: 910163b6b09685a35c4ebbc52b66d1bfbbe39fc5
   React-RCTAnimation: 9a883bbe1e9d2e158d4fb53765ed64c8dc2200c6
   React-RCTBlob: 39cf0ece1927996c4466510e25d2105f67010e13
@@ -802,24 +802,24 @@ SPEC CHECKSUMS:
   React-RCTText: 1b6773e776e4b33f90468c20fe3b16ca3e224bb8
   React-RCTVibration: 4d2e726957f4087449739b595f107c0d4b6c2d2d
   ReactCommon: a0a1edbebcac5e91338371b72ffc66aa822792ce
-  RNDateTimePicker: 3a1783e63fcb6e2b162b08a4321db030d23accff
+  RNDateTimePicker: a84d2b439666c9243d42a8396c588d4d1d36e3f3
   RNFBApp: 43a062dc6a1a88edd934d2229d1ba64f25381598
   RNSentry: 6b46b6fc1d715a378fbaa5d7d43bc9ce99b500e5
   RNSVG: ce9d996113475209013317e48b05c21ee988d42e
   Sentry: 388c9dc093b2fd3a264466a5c5b21e25959610a9
-  UMAppLoader: 98b77b58fac12abab963f79f762ceef008c50505
-  UMBarCodeScannerInterface: e1299aafa3bede8ba674e84199e9f319a0195cad
-  UMCameraInterface: 68da359d6fe7423d7b114b96ea0df268a84da39f
-  UMConstantsInterface: 2569e83b7f70a0844737e1f6c2dfd1bc0c8a34d6
-  UMCore: ef0c10289950b51d0c1dae75c37fba116f2327ac
-  UMFaceDetectorInterface: b0f3be22a524053f29d71c50bfdf51dc9553f7aa
-  UMFileSystemInterface: 29a555288008c9fa9be7d781c84f58ef51d41de6
-  UMFontInterface: 5ece8ea0e3c5753a87a558dcd7e79dcf38a6cab6
-  UMImageLoaderInterface: c440e3cd54c060cb2f9a278a923e28c6fe0ec116
-  UMPermissionsInterface: d6be0aea537754c92648e3d7100c58dd72c01142
-  UMReactNativeAdapter: 5973dd1e4374cd1db4549b6bc714ccbbde9615c0
-  UMSensorsInterface: 7bc27058e00af99695e16f74c092582698be964d
-  UMTaskManagerInterface: 8b337c26b828b3e50d26776966981f4cb2ee76e4
+  UMAppLoader: 61049c8d55590b74e9ae1d5429bf68d96b4a2528
+  UMBarCodeScannerInterface: e5e4c87797d3d01214e25cd1618866caf5d4f17f
+  UMCameraInterface: 415ac060034edecacdbbaa739c223e3f276e0056
+  UMConstantsInterface: 1a52f2d884c95e8829439da13e36b7669a1a8fb4
+  UMCore: d98083b522b08c0a8ba3992bc263c624ae5d887c
+  UMFaceDetectorInterface: 67c6c82451338da01a4bc00ec46365a2a8ea9057
+  UMFileSystemInterface: 303d696ede28102a7e11d111808bd2ed2c5eb62f
+  UMFontInterface: 6edf1ee8bc55d2030766f8cf0a7b20a5d5a913b0
+  UMImageLoaderInterface: 9cdbf3bab6a513bddd88505cb2340fe02d6a11c0
+  UMPermissionsInterface: 019170ad655f464e3f8d23d2a8bcbda2e645cde4
+  UMReactNativeAdapter: 538efe92e781b5d7678cf95b34c46f2d0989a557
+  UMSensorsInterface: cb5bf31d52c4349f0ff9e3c049bbe4df0d80d383
+  UMTaskManagerInterface: 80653f25c55d9e6d79d6a0a65589fa213feaee11
   Yoga: 7740b94929bbacbddda59bf115b5317e9a161598
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
```

**Comment by MarkDavies0** [comment]
OK I pushed my podfile (which had changes) and it looks like they might be in sync now, can you check against yours

**Comment by rlnsy** [comment]
Yep! Looks good.

**Review by rlnsy — APPROVED** [review]

**Comment by wynnset** [comment]
@rlnsy are you still looking at this or can it be merged in? I'm asking because it's still assigned to you.

**Comment by rlnsy** [comment]
Nope LGTM

**Review by wynnset — APPROVED** [review]

**Pull request merged**